src/zconfig.c

```diff
@@ -939,6 +961,47 @@ zconfig_print (zconfig_t *self)
 //  --------------------------------------------------------------------------
 //  Self test of this class
 
+static void zconfig_test_save_load_long_value (bool verbose)
+{
+    if (verbose) printf("\n%s\n", "zconfig_test_save_load_long_value");
+
+    zconfig_t *root = zconfig_new ("root", NULL);
+    assert (root);
+    zconfig_t *item = zconfig_new ("item", root);
+    assert (item);
+
+    // see zconfig_chunk_load(), predefined char *cur_line[1024 + 1];
+    const int sz0 = 1000, sz1 = 1048;
```

Comment:
Would be better if constants are linked to buffer size in `zconfig_chick_load()`.